verify/cli.go

```diff
@@ -20,6 +21,9 @@ var (
 	G2PathFlagName                   = withFlagPrefix("g2-path")
 	G2TrailingPathFlagName           = withFlagPrefix("g2-path-trailing")
 	CachePathFlagName                = withFlagPrefix("cache-path")
+
+	// rollup related flags
+	RollupBlobInclusionWindowFlagName = withFlagPrefix("rollup-blob-inclusion-window")
```

Comment:
don't think this should be here.. since `verify/` is v1 only, but we also need to do this check for v2. Maybe should have a `rollup` category?

Reply:
[de36b93](https://github.com/Layr-Labs/eigenda-proxy/pull/378/commits/de36b938fec11748571d35abd501ef09bf5a37d1)